README.md

```diff
@@ -1,7 +1,62 @@
-# codeartifact-workflows
-Public re-usable GitHub workflows for working with AWS CodeArtifact
+# python-lib-workflows
+Public re-usable GitHub workflows for working with python libraries which autorelease with bumpversion and publish to AWS CodeArtifact
 
 # Usage
+These reusable workflows are designed to run in sequence and take a `CUSTOM_GITHUB_TOKEN` to use in place of the standard CI token. This is because actions like commits, tags, releases, etc. performed by the CI token do not trigger other workflows as you would expect.
+
```

Comment:
## Support
### Contacting the team(s)
TODO - who maintains this?  e.g., Is it in the codeowners file?

# Developer Info

Reply:
Not going to add this since the repo is public.